src/commands/view/OpenManagers.js

```diff
@@ -0,0 +1,116 @@
+import Backbone from 'backbone';
+const StyleManager = require('style_manager');
+const $ = Backbone.$;
+
+module.exports = {
+  run(em, sender) {
+    this.sender = sender;
+    this.em = em;
+    if (!this.$tmCn || !this.smCn) {
+      this.prepareTm();
+      this.prepareSm();
+      this.listenTo(this.target, 'component:toggled', this.toggleManagers);
+    }
+    this.toggleManagers();
+  },
+  prepareTm() {
+    const em = this.em;
+    const config = em.Config;
+    const pfx = config.stylePrefix;
+    const tm = em.TraitManager;
+    var panelC;
+
+    if (!this.$tmCn) {
+      var tmView = tm.getTraitsViewer();
+      var confTm = tm.getConfig();
+      this.$tmCn = $('<div></div>');
+      this.$tmCn2 = $('<div></div>');
+      this.$tmCn.append(this.$tmCn2);
+      this.$tmHeader = $('<div>').append(
+        `<div class="${confTm.stylePrefix}header">${confTm.textNoElement}</div>`
+      );
+      this.$tmCn.append(this.$tmHeader);
+      this.$tmCn2.append(
+        `<div class="${pfx}traits-label">${confTm.labelContainer}</div>`
+      );
+      this.$tmCn2.append(tmView.render().el);
+      var panels = em.Panels;
+
+      if (!panels.getPanel('views-container'))
+        panelC = panels.addPanel({ id: 'views-container' });
+      else panelC = panels.getPanel('views-container');
+
+      panelC
+        .set('appendContent', this.$tmCn.get(0))
+        .trigger('change:appendContent');
+
+      this.target = em.getModel();
+      this.listenTo(this.target, 'component:toggled', this.toggleTm);
+    }
+  },
```

Comment:
this is really up to you, but I don't know that we really want to stick with his naming conventions. It's pretty much illegible, but if all the code looks like this (which I happen to know it does), it may make sense to leave it this notation system.